src/main/java/jewellery/inventory/JewelleryInventoryApplication.java

```diff
@@ -6,8 +6,7 @@
 @SpringBootApplication
 public class JewelleryInventoryApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(JewelleryInventoryApplication.class, args);
-	}
-
+    public static void main(String[] args) {
+        SpringApplication.run(JewelleryInventoryApplication.class, args);
+    }
```

Comment:
Same here, you haven't made any changes here but it pops up as a change. Make sure we don't have obsolete changes.

Reply:
I was messing with this file while testing but then reverted the changes I guess the formatting is the issue.

Reply:
This is still showing up as change. Try to checkout the test version of it by doing `git checkout origin/test src/main/java/jewellery/inventory/JewelleryInventoryApplication.java`

Reply:
Fixed